lib/AnonymizationProxy/anonymizer.js

```diff
@@ -0,0 +1,56 @@
+const emailRegex = /([a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,6})/g;
+const nameRegex = /\b([A-Z][a-z]+\s[A-Z][a-z]+)\b/g;
```

Comment:
If the body has any words matching this pattern, will you anonymize? It does not seem right